scripts/csv_to_parquet.py

```diff
@@ -0,0 +1,22 @@
+import os.path
+import sys
+
+import pyarrow.csv as pv
+import pyarrow.parquet as pq
+
+args = sys.argv
+
```

Comment:
Please add a comment explaining a bit the Why, behind this script.

Reply:
No problem